config/lobby/db/001_create_tables

```diff
@@ -1,94 +1,58 @@
-create table bad_words (
-    word character varying(40) not null
+CREATE TABLE ta_users (
+    username character varying(40) NOT NULL PRIMARY KEY,
+    password character(34),
+    bcrypt_password character(60),
+    CHECK (password IS NOT NULL OR bcrypt_password IS NOT NULL),
+    email character varying(254) NOT NULL,
+    joined timestamp with time zone DEFAULT CURRENT_TIMESTAMP,
+    lastlogin timestamp with time zone DEFAULT CURRENT_TIMESTAMP,
+    admin integer NOT NULL DEFAULT 0
 );
+ALTER TABLE ta_users owner TO postgres;
 
-
-alter table bad_words owner to postgres;
-
-
-create table banned_ips (
-    ip character varying(40) not null,
-    ban_till timestamp without time zone
+CREATE TABLE bad_words (
+    word character varying(40) NOT NULL PRIMARY KEY
 );
+ALTER TABLE bad_words owner TO postgres;
 
 
-alter table banned_ips owner to postgres;
-
-
-create table banned_macs (
-    mac character varying(40) not null,
-    ban_till timestamp without time zone
+CREATE TABLE banned_ips (
+    ip character varying(40) NOT NULL PRIMARY KEY,
+    ban_till timestamp with time zone
 );
+ALTER TABLE banned_ips owner TO postgres;
 
 
-alter table banned_macs owner to postgres;
-
-
-create table banned_usernames (
-    username character varying(40) not null,
-    ban_till timestamp without time zone
+CREATE TABLE banned_macs (
+    mac character varying(40) NOT NULL PRIMARY KEY,
+    ban_till timestamp with time zone
 );
+ALTER TABLE banned_macs owner TO postgres;
 
 
-alter table banned_usernames owner to postgres;
-
-create table muted_ips (
-    ip character varying(40) not null,
-    mute_till timestamp without time zone
+CREATE TABLE banned_usernames (
+    username character varying(40) NOT NULL PRIMARY KEY REFERENCES ta_users ON DELETE CASCADE ON UPDATE CASCADE,
+    ban_till timestamp with time zone
 );
+ALTER TABLE banned_usernames owner TO postgres;
 
 
-alter table muted_ips owner to postgres;
-
-create table muted_macs (
-    mac character varying(40) not null,
-    mute_till timestamp without time zone
+CREATE TABLE muted_ips (
+    ip character varying(40) NOT NULL PRIMARY KEY,
+    mute_till timestamp with time zone
```

Comment:
Keeping with the theme of skipping time zone complexity altogether:
1. can we default time zone to UTC and skip any mention of time zone in the code?
2. instead maybe use an epoch milli time stamp stored as a number?

I think for now it may be better to keep this branch focused on the bcrypt password updates and let's tackle time zone updates on their own.